### DISCUSSION TITLE
🖐️ Does Dagger support pipeline operation cache invalidation yet?

### DISCUSSION THREAD

**Discussion by dylanrhysscott** [dagger/dagger#12316] [Help]
### What happened? What did you expect to happen?

👋 Have been playing with dagger by building my own module for running Terraform and other scripts. Am running into a weird issue where previous deploy / destroy runs have been cached to the point where Dagger just shows the previous successful run rather than deploying infrastructure again even though it no longer exists. In essence making multiple runs quite tricky. While searching for a solution I found this in the archive as a workaround to force fresh exec runs https://archive.docs.dagger.io/0.9/cookbook/#invalidate-cache. Does Dagger support invalidating the pipeline operations cache yet so this workaround isn't needed?

> This is a temporary workaround until cache invalidation support is officially added to Dagger.

Thanks

**Comment by helderco** [comment]
> Does Dagger support invalidating the pipeline operations cache yet so this workaround isn't needed?

No, no explicit cache invalidation op yet.
